src/services/FileList.js

```diff
@@ -34,7 +34,6 @@ export default async function(user, path) {
 		method: 'PROPFIND',
 		url: generateRemoteUrl(`dav/files/${user}${path}`),
 		headers: {
-			requesttoken: OC.requestToken,
```

Comment:
fun fact: this version didn't have the bug `nextcloud-axios` had as it ready the request token anew for every request :man_shrugging: